ChangeLog.md

```diff
@@ -1,5 +1,264 @@
 # Changelog for haskell-language-server
 
+## 1.1.0
+
+Haskell Language Server 1.1.0 has finally come! Many thanks to all contributors -- since the last release, we have merged over 100 PRs! As always, there are many internal bug fixes and performance improvements in ghcide. Apart from that,
+
+* Wingman gets a comprehensive evolution, thanks to @isovector for this epic work!
+* hlint plugin not working in some cases gets fixed
+* scary but unharmful log message "haskell-lsp:incoming message parse error" is fixed in `lsp-1.2`
+* eval plugin now supports `it` variable, like GHCi
+* verbose message "No cradle found for ... Proceeding with implicit cradle" is gone
+* type lenses plugin now has its custom config `mode` (enum) to control its working mode:
+  * `always`: always displays type signature lenses of global bindings
+
+  * `exported`: similar to `always`, but only displays for exported global bindings
+
+  * `diagnostics`: follows diagnostic messages produced by GHC
+* top-level LSP option `completionSnippetsOn` and `maxNumberOfProblems` are deprecated
+* completions plugin now has its custom config:
+  * `autoExtendOn` (boolean): whether to enable auto extending import lists
+  * `snippetsOn` (boolean): wheter to enable completion snippets, taking the place of `completionSnippetsOn`
+* LSP symbols of typeclass and type families are more appropriate
+* test suite of plugins are reorganized, which no longer need to be run with `test-server` executable
+* two new packages `hls-test-utils` and `hls-stylish-haskell-plugin` are extracted
+
+### Pull requests merged for 1.1.0
+
+\- Wingman: "Destruct all" only on ADTs
```

Comment:
Is it intentional that all of these have a leading `\`? It means they display all on one line in GitHub.

Reply:
Ah, they were added by the editor accidently. Thank you for pointing out!